src/types.ts

```diff
@@ -208,6 +210,20 @@ export interface VoteWeight {
 	readonly delegates: ReadonlyArray<DelegateWeight>;
 }
 
+export interface GenesisBlockGenerateInput {
```

Comment:
Can we not directly import the type/interface from the SDK instead of manually defining it here?